bin/utils/subkey/src/main.rs

```diff
@@ -16,7 +16,7 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! Subkey utility, based on node_runtime.
+//! Subkey utility, based on kitchensink_runtime.
```

Comment:
```suggestion
//! Subkey utility
```

I don't see any link between this utility and the node/kitchensink runtime